examples/dvc-mltrace/02_summary_stats.py

```diff
@@ -0,0 +1,25 @@
+import pandas as pd
+from mltrace import create_component, register
+
+
+@register(component_name="summary", input_vars=["input_filepath"])
```

Comment:
Maybe include an `output` here so there is some output id to trace? Like possibly you can dump the summary stats to some file and log that filename in `output_vars`.

Reply:
component now writes output of `df.describe()` to a csv instead of printing information.